libsel4/include/sel4/bootinfo_types.h

```diff
@@ -28,7 +28,8 @@ enum seL4_RootCNodeCapSlots {
     seL4_CapSMMUCBControl       = 13, /* global SMMU CB controller cap, null cap if not supported */
     seL4_CapInitThreadSC        = 14, /* initial thread's scheduling context cap, null cap if not supported */
     seL4_CapSMC                 = 15, /* global SMC cap, null cap if not supported */
-    seL4_NumInitialCaps         = 16
+    seL4_CapSBI                 = 16, /* global SBI cap, null cap if not supported */
+    seL4_NumInitialCaps         = 17
```

Comment:
```suggestion
    seL4_CapSBI                 = seL4_CapSMC, /* global SBI cap, null cap if not supported */
    seL4_NumInitialCaps         = 16
```
Because it's the same thing with a different name.